dist/21-526EZ-schema.json

```diff
@@ -1634,8 +1629,7 @@
       "items": {
         "type": "object",
         "required": [
-          "treatmentCenterName",
-          "treatmentCenterType"
```

Comment:
Looks like there's some code referencing this in the front end, but it looks like it's removing this property, so it shouldn't break or anything.

Reply:
That's a good point, I can take out that `omit` when I open a PR to update the version number.